### PR TITLE
Run-once pod duration: remove flag from plugin config

### PR DESCRIPTION
**Pull request by csrwng**
Sets up the config to be similar to ClusterResourceOverride plugin, nil config means disabled.

### CODE REVIEW

**Comment by csrwng** [comment]
@deads2k @sosiouxme ptal

**Comment by sosiouxme** [comment]
LGTM

File: pkg/quota/admission/runonceduration/admission.go [hunk 1]
```diff
@@ -30,14 +30,14 @@ func init() {
 }
 
 func readConfig(reader io.Reader) (*api.RunOnceDurationConfig, error) {
```
Comment (deads2k) [inline pkg/quota/admission/runonceduration/admission.go]: This method is getting repeated a lot.  It looks like the intent is `ReadYAML` as opposed to the existing `ReadYamlInto` behavior (yeah, poorly named).

How about doing that refactor.

**Comment by csrwng** [comment]
@deads2k added another commit ... is that what you were asking for?

**Comment by csrwng** [comment]
updated ReadYAML function

**Comment by deads2k** [comment]
lgtm

**Comment by csrwng** [comment]
rebased and squashed
[merge]

**Pull request merged**